app/models/game.rb

```diff
@@ -1,5 +1,51 @@
 class Game < ApplicationRecord
   validates :name, presence: true
+  after_create :populate_board!
```

Comment:
Using an `after_create` can make testing very hard/brittle later because it's not obvious where the pieces came from.

Take a look at this pattern 😄 https://gist.github.com/mscoutermarsh/2f6a04729a9136ccfb7bf940568cbae4

Reply:
Hi Mike, so after I move this code to game_controller and follow your pattern, my test for all the pieces' positions started to fail, and I have to change it to:
"    game = FactoryBot.create(:game)
      game.populate_board
" for it to work. Is that normal?